StripeUICore/StripeUICore/Source/Categories/UIColor+StripeUICore.swift

```diff
@@ -123,13 +123,13 @@ import UIKit
 
 // MARK: - Helpers
 
-private extension UIColor {
+extension UIColor {
 
     /// Transforms the brightness and returns the resulting color.
     ///
     /// - Parameter transform: A block for transforming the brightness.
     /// - Returns: Updated color.
-    func byModifyingBrightness(_ transform: @escaping (CGFloat) -> CGFloat) -> UIColor {
+    fileprivate func byModifyingBrightness(_ transform: @escaping (CGFloat) -> CGFloat) -> UIColor {
```

Comment:
I don't like the use of `fileprivate` here, this is caused by the `NoAccessLevelOnExtensionDeclaration` rule, which is good for `public` extensions, but I don't like it for `private` ones.
Thoughts?

Reply:
Eh, it's not a choice I would make without a linter, but it looks like it *is* only used in this file.

When it adds public to these (e.g. `public func lighten()`), I noticed that it's not adding the `@_spi()`. Is that okay? Will these still be marked as SPI if the outer extension is marked as SPI?

Reply:
> Eh, it's not a choice I would make without a linter, but it looks like it is only used in this file.

Might be the only place in this project, but it happens in other places too.

> When it adds public to these (e.g. public func lighten()), I noticed that it's not adding the @_spi(). Is that okay? Will these still be marked as SPI if the outer extension is marked as SPI?

Very good point, I did not think of that, but I just checked and it works correctly.

<img width="1342" alt="Screen Shot 2022-11-10 at 11 59 19 AM" src="https://user-images.githubusercontent.com/114543737/201194125-622afb3f-8186-479a-b18b-7b67c3f72284.png">